repos/system_upgrade/common/actors/checkdetecteddevicesanddrivers/libraries/checkdddd.py

```diff
@@ -0,0 +1,178 @@
+from leapp import reporting
+from leapp.libraries.common.config.version import get_target_major_version, get_source_major_version
+from leapp.libraries.stdlib import api
+from leapp.models import DetectedDeviceOrDriver, Report
+from enum import IntEnum
+from collections import defaultdict
+
+
+class MessagingClass(IntEnum):
+    Unknown = 0
+    Drivers = 1
+    Devices = 2
+    CPUs = 3
+
+
+def create_inhibitors(inhibiting_entries):
+    if not inhibiting_entries:
+        return
+
+    drivers = inhibiting_entries.get(MessagingClass.Drivers)
+    if drivers:
+        reporting.create_report([
+            reporting.Title(
```

Comment:
I understand that this is too broad but does the HW team think about preparing some generic guide about what to do / where to look for alternatives in case users has been affected by the removed entity?

Reply:
I don't know about that, TBH I haven't checked for that